manifests/00-crd-extension-console-link.yaml

```diff
@@ -54,3 +54,17 @@ spec:
               type: string
               description: Determines which dropdown menu the link is added to (HelpMenu, UserMenu)
               pattern: ^(HelpMenu|UserMenu)$
+            applicationMenu:
+              type: object
+              description: Object that holds application menu defaults if the location is set to ApplicationMenu
+              required:
+                - section
+                - imageURL
+              properties:
+                section:
+                  type: string
+                  description: The section of the application menu in which the link should appear
+                imageURL:
+                  type: string
+                  description: The URL for the icon used in front of the link in the application menu.  The URL must be an HTTPS URL or a Data URI. The image should be square and will be shown at 24x24 pixels.
```

Comment:
@rhamilto does this default to anything?

Reply:
If it does, we don't need to list it as required.

Reply:
@jeff-phillips-18 

I don't think we should make it required. I'd guess it just doesn't show an icon.

Reply:
agree, lets dont make the `imageURL` required.

Reply:
Sounds good.